appveyor-nightly.yml

```diff
@@ -65,6 +65,8 @@ test_script:
 after_test:
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, ubuntu18]"
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, centOS7]"
+  - rake "download_portable[develop]"
+  - appveyor PushArtifact "C:/projects/ospsuite-r/pksim_minimal.zip"
```

Comment:
I don't quite understand the procedure. 
I thought, the required PK-Sim DLLs would be added to the R package (in the same way as the OSPSuite.Core DLLs are added). Instead, they are combined into **pksim_minimal.zip** and pushed as an additional artifact on AppVeyor?

Reply:
The first idea was to specify the portable PKsim associated with the build so that it could be downloaded as artefacts and even used

Reply:
Right now we create a folder that has no meaning whatsoever
Then, if this is still too cumbersome, we said we would create another package with all the DLLs. 

I don't think we said we would update our standard package with this

Reply:
>we said we would create another package with all the DLLs.
I don't think we said we would update our standard package with this

Fine for me - less testing :)